scripts/redis_crash_handler.py

```
@@ -0,0 +1,893 @@
+        response = requests.get(
+            f"{self.base_url}/select/logsql/query",
+            params=params,
+            auth=self.auth,
+            timeout=60,
+            verify=False
+        )
+        
+        # Check for authentication issues
+        if response.status_code == 401:
+            print("❌ ERROR: Authentication failed (401 Unauthorized)")
+            raise ValueError(f"Authentication failed for VictoriaLogs. Check VMAUTH_USERNAME and VMAUTH_PASSWORD.")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)